lib/sys/compiler.d

```diff
@@ -0,0 +1,58 @@
+// Written in the D programming language.
+
```

Comment:
I wish we'd break away from this. What's the rationale for it ?

Reply:
IIRC, it's so that search engines more easily identify the code as being D code.

Personally, I do it everywhere, since it's not a big deal, and maybe it really does help with that sort of thing, though I would guess that it was more likely to have helped when D was started than now.